sirepo/sim_api/jupyterhublogin.py

```diff
@@ -21,6 +21,8 @@
 
 _HUB_USER_SEP = "-"
 
+_INVALID_USER_NAME_CHARS = re.compile(r"[^a-z0-9]+")
```

Comment:
A comment here might be nice, because this includes `-` which is a valid username char but not for this purpose.

Reply:
[3a9f0d1](https://github.com/radiasoft/sirepo/pull/7027/commits/3a9f0d1399801354cea9e77a661893218636e2dd)